R/symptome_compartment_class.R

```diff
@@ -0,0 +1,226 @@
+#' An SEIR model for compartments of different symptoms
+#'
+#' Note:
+#' 1) Sum of poulation equals to 1 at any time point.
+#' An error will be reported the sum of initial population is not equal to 1.
+#' 2) At this stage, we only consider disease-caused death cases
+#' calculating mortality.
+#'
+#' model structure:
+#'    ---------------------------(omega)---------------------------------
+#'    |                                                                 |
+#'    |                                                                 |
+#'    --> S --(lam)--> E --(gamma) ------(1-ihr)-------> I ---(nu.i)--> R <--
```

Comment:
I'm not sure I understand notation here. What's omega? What does CM or HA mean? I suspect it may be easier to represent things in equation form...

Reply:
It is just to make a simple flowchart-like diagram for myself to check which population groups and parameters are included. Docstrings for the params and groups are further down in the code.  I meant to remove the part when the class is finished, but I agree it's better to represent by equations in docstrings. Will do it soon.

Reply:
I agree with Ben. I think this might be a great graph to have in the vignette that will go with this class though!